tests/test_pipeline.py

```diff
@@ -127,39 +124,18 @@ def test_train_mahalanobis_advex(backdoor_classifier_path, tmp_path):
         detectors.DebugQuantumEntropyConfig,
     ],
 )
-@pytest.mark.parametrize("train_on_clean", [False, True])
-def test_train_statistical_backdoor(
-    backdoor_classifier_path, tmp_path, detector_type, train_on_clean
-):
+def test_train_statistical_backdoor(backdoor_classifier_path, tmp_path, detector_type):
     cfg = train_detector_conf.Config(
         task=tasks.backdoor_detection.DebugBackdoorDetection(
             path=backdoor_classifier_path,
             backdoor=data.CornerPixelBackdoor(),
```

Comment:
```suggestion
```
Backdoor is no longer an argument to tasks. This is the same for all the below tests as well.